runner/compute-driver/local/compute-execute

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+# local/compute-execute -- Executes a process locally using all available processors
+# $ compute-execute input_sql=... command=... output_relation=...
+#
+# To limit the number of parallel processes, set the DEEPDIVE_NUM_PROCESSES
+# environment or the 'deepdive.computers.local.num_processes' in
+# computers.conf:
+# $ export DEEPDIVE_NUM_PROCESSES=2
+# $ compute-execute input_sql=... command=... output_relation=...
+##
+set -euo pipefail
+
+: ${DEEPDIVE_PREFIX_TABLE_TEMPORARY:=dd_tmp_} ${DEEPDIVE_PREFIX_TABLE_OLD:=dd_old_}
+
+# load compute configuration
+eval "$(jq2sh <<<"$DEEPDIVE_COMPUTER_CONFIG" \
+    num_processes='.num_processes' \
+    #
+)"
+# respect the DEEPDIVE_NUM_PROCESSES environment
+num_processes=${DEEPDIVE_NUM_PROCESSES:-${num_processes:-$(
+        # detect number of processor cores
+        nproc=$(
+            # Linux typically has coreutils which includes nproc
+            nproc ||
+            # OS X
+            sysctl -n hw.ncpu ||
+            # fall back to 1
+            echo 1
+        )
+        if [[ $nproc -gt 1 ]]; then
+            # leave one processor out
+            let nproc-=1
+        elif [[ $nproc -lt 1 ]]; then
+            nproc=1
+        fi
+        echo $nproc
+    )}}
+
+# declare all input arguments
+declare -- "$@"
+
+# show configuration
+echo "Executing with the following configuration:"
+echo " num_processes=$num_processes"
+
+# XXX there are conditional branches below depending on whether input_sql
+# and/or output_relation is given, to support four use cases:
+# 1) executing command while streaming data from/to the database
+# 2) input-only command which has no output to the database and streams from the database
+# 3) output-only command which has no input from the database and streams to the database
+# 4) database-independent command which simply runs in parallel
+
+# prepare a temporary output table when output_relation is given
+if [[ -n $output_relation ]]; then
+    # some derived values
+    output_relation_tmp="${DEEPDIVE_PREFIX_TABLE_TEMPORARY}${output_relation}"
+
+    # show configuration
+    echo " output_relation_tmp=$output_relation_tmp"
+    echo
+
+    # use an empty temporary table as a sink instead of TRUNCATE'ing the output_relation
+    deepdive-initdb "$output_relation"
+    deepdive-sql "
+        DROP TABLE IF EXISTS $output_relation_tmp;
+        CREATE TABLE $output_relation_tmp (LIKE $output_relation);
```

Comment:
Maybe add `INCLUDING ALL` to make sure all other bells and whistles are retained?